FlowCrypt/Functionality/Services/Key Services/KeyService.swift

```diff
@@ -34,26 +34,19 @@ final class KeyService: KeyServiceType {
     }
 
     func retrieveKeyDetails() -> Result<[KeyDetails], KeyServiceError> {
```

Comment:
The difference between `retrieveKeyDetails` and `getPrivateKeys` is not obvious from the name. Let's rename it:
 - `retrieveKeyDetails` -> `getPrvKeyDetails`
 - `getPrivateKeys` -> `getPrvKeyInfo`

Still not so obvious, but at least it highlights the different formats.

Then, please add a comment to highlight what is each method for, why do we have two similar methods.